azure-iot-device/azure/iot/device/provisioning/pipeline/provisioning_pipeline.py

```diff
@@ -207,7 +207,7 @@ def on_complete(op, error):
             if error:
                 callback(error=error, result=None)
             else:
-                callback(result=op.registration_result)
+                callback(error=None, result=op.registration_result)
 
```

Comment:
I don't really understand the need for this change

Reply:
I did it for test consistency, but I agree that it's not necessary

---
In reply to: [394562682](https://github.com/Azure/azure-iot-sdk-python/pull/499#discussion_r394562682) [](ancestors = 394562682)